wpimath/src/main/native/include/wpi/math/kinematics/Kinematics.hpp

```diff
@@ -18,7 +19,8 @@ namespace wpi::math {
  * Inverse kinematics converts a desired chassis speed into wheel speeds whereas
  * forward kinematics converts wheel speeds into chassis speed.
  */
-template <typename WheelSpeeds, typename WheelPositions>
+template <typename WheelSpeeds, typename WheelPositions,
+          typename WheelAccelerations>
```

Comment:
Why is the order different between Java and C++? (Java uses S, A, P while C++ uses WheelSpeeds, WheelPositions, WheelAccelerations (S, P, A))